src/features/class-fields.md

```diff
@@ -191,14 +191,14 @@ class Cat extends Animal {
 
 <feature-support chrome="72 /blog/v8-release-72#public-class-fields"
                  firefox="partial https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/69#JavaScript"
-                 safari="no"
+                 safari="partial"
```

Comment:
Are there any links and/or version numbers?

Reply:
There are many bugzilla issues but only public class fields are available under --useClassFields JSC flag in Technology Preview for now.

Reply:
Hmm so they don't have a meta-bug to track all of these? In that case let's link to one for public class fields where the flag is documented.

Reply:
(same for other features / browsers in the PR)